atlas-shopify-blueprint/configuring-shopify.mdx

```diff
@@ -0,0 +1,29 @@
+---
+slug: /atlas-shopify-blueprint/configuring-shopify
+title: Atlas Shopify Blueprint - Configuring Shopify
+description: Configure the Shopify Blueprint with Shopify Storefront API credentials.
+---
+
+## Configuring Shopify
+
+In order to list products from your **Shopify** store in the Blueprint once it has been deployed, you need to add your Shopify Access Token and Shopify GraphQL URL as Environment Variables.
```

Comment:
We can at least get this bit more aligned with Shopify's terminology, ahead of [TITAN-385](https://wpengine.atlassian.net/browse/TITAN-385):

```suggestion
In order to list products from your **Shopify** store in the Blueprint once it has been deployed, you need to add your [Shopify Headless Public Access Token](https://shopify.dev/docs/api/usage/authentication#getting-started-with-public-access) and [Shopify GraphQL URL](https://shopify.dev/docs/api/storefront#endpoints) as Environment Variables.
```